av.c

```diff
@@ -99,7 +99,7 @@ Perl_av_extend_guts(pTHX_ AV *av, SSize_t key, SSize_t *maxp, SV ***allocp,
     PERL_ARGS_ASSERT_AV_EXTEND_GUTS;
 
     if (key < -1) /* -1 is legal */
-        Perl_croak(aTHX_
+        croak(
             "panic: av_extend_guts() negative count (%" IVdf ")", (IV)key);
```

Comment:
This and several others could now have the argument(s) pulled up on the same line as the function name.

Reply:
Yeah I did that in a few places, but I can probably do it in more